pyintegration/deephaven2/uri.py

```diff
@@ -0,0 +1,87 @@
+#
+#   Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+#
+""" Tools for resolving URIs into objects including Tables. """
+
+import inspect
+from typing import Any, Union
+
+import jpy
+
+from deephaven2 import DHError
+from deephaven2._wrapper_abc import JObjectWrapper
+
+_JResolveTools = jpy.get_type("io.deephaven.uri.ResolveTools")
+
+
+def _is_direct_initialisable(cls):
+    funcs = inspect.getmembers(cls, inspect.isfunction)
+    init_funcs = [func for name, func in funcs if name == "__init__"]
+    if init_funcs:
+        init_func = init_funcs[0]
+        sig = inspect.signature(init_func)
+        if len(sig.parameters) == 2:
+            _, param_meta = list(sig.parameters.items())[1]
+            if param_meta.annotation == "jpy.JType":
+                return True
+
+    return False
+
+
+def _find_deephaven_wrappers():
+    """Returns a set of Python classes that wrap the Deephaven Java ones and must be
+    directly initialisable."""
+    import deephaven2
```

Comment:
How do we remember to change this when dh2 becomes the standard?

Reply:
if the deephaven2 name no longer exists, the unit tests will catch it if we miss it in our refactoring effort.